corehq/apps/receiverwrapper/views.py

```diff
@@ -39,13 +40,15 @@ def _process_form(request, domain, app_id, user_id, authenticated,
         except:
             meta = {}
 
-        notify_exception(None, "Received a submission with POST.keys()", {
+        details = {
             "domain": domain,
             "app_id": app_id,
             "user_id": user_id,
             "authenticated": authenticated,
             "form_meta": meta,
-        })
+        }
+        log_counter(MULTIMEDIA_SUBMISSION_ERROR_COUNT, details)
+        notify_exception(None, "Received a submission with POST.keys()", details)
```

Comment:
sweet. is there a chance we should just completely remove the email notification or you still think that's important?